Composer/packages/client/src/components/ErrorPopup/ErrorPopup.tsx

```diff
@@ -62,7 +61,7 @@ export const ErrorPopup = (props: ErrorPopupProps) => {
     }
   }, [props.error]);
 
-  const title = props.error ? formatErrorTitle(props.error) : '';
+  const title = props.error && props.error.summary ? formatErrorTitle(props.error.summary) : 'Error';
```

Comment:
Consider making this formatErrorTitle(props?.error?.summary) then in formatErrorTitle if !summary then return formatMessage('Error')